resources/services/coins/tickers.go

```diff
@@ -52,7 +52,7 @@ func fetchTickers(ctx context.Context, meta schema.ClientMeta, parent *schema.Re
 	startDate := cl.StartDate
 	key := fmt.Sprintf(stateKeyTpl, *c.ID)
 	if cl.Backend != nil {
-		value, err := cl.Backend.Get(ctx, key, cl.ID())
+		value, err := cl.Backend.GetKey(ctx, key)
```

Comment:
IMO you could just use state.NopClient & never perform the `if cl.Backend != nil` check

Reply:
still here